scripts/pricefeeds/pricefeeds.py

```diff
@@ -299,12 +305,42 @@ def derive_prices(feed):
             print("[Warning] No market route found for %s. Skipping price" % asset)
             continue
 
-        # price convertion to "price for one BTS" i.e.  base=*, quote=core_symbol
-        price_result[asset] = {"mean"    : price_mean,
+        # price conversion to "price for one BTS" i.e.  base=*, quote=core_symbol
+        price_result[asset]['BTS'] = {"mean"    : price_mean,
                                "median"  : price_median,
                                "weighted": price_weighted,
                                "std"     : price_std * 100,  # percentage
-                               "number"  : len(assetprice)}
+                               "number"  : len(assetprice),
+                               "quote"   : blockchain_feed_quote[asset],
+                               }
+
+        assetvolume = [v for v in volume[asset][quote_asset]]
+        assetprice  = [p for p in price[asset][quote_asset]]
+        if len(assetvolume) > 1 :
+            price_median = statistics.median(price[asset][quote_asset])
+            price_mean   = statistics.mean(price[asset][quote_asset])
+            price_weighted = num.average(assetprice, weights=assetvolume)
+            price_std      = weighted_std(assetprice, assetvolume)
+            if price_std > 0.1 :
+                print("Asset %s shows high variance in fetched prices!" % (asset))
+        elif len(assetvolume) == 1:
+            price_median   = assetprice[0]
+            price_mean     = assetprice[0]
+            price_weighted = assetprice[0]
+            price_std      = 0
+            print("[Warning] Only a single source for the %s price could be identified" % asset)
+        else :
+            print("[Warning] No market route found for %s. Skipping price" % asset)
+            continue
+        # price conversion to "price for one quote_asset" i.e.  base=*, quote=quote_asset
+        price_result[asset]["quote"] = {"mean"    : price_mean,
+                               "median"  : price_median,
+                               "weighted": price_weighted,
+                               "std"     : price_std * 100,  # percentage
+                               "number"  : len(assetprice),
+                               "quote"   : blockchain_feed_quote[asset],
+                               }
+
```

Comment:
What's the difference to lines 288-306? I don't see how you change from BTS to short backing asset